docs/Language Feature Status.md

```diff
@@ -10,19 +10,20 @@ efforts behind them.
 
 | Feature | Branch | State | Developer | Reviewer | LDM Champ |
 | ------- | ------ | ----- | --------- | -------- | --------- |
+| [List patterns](https://github.com/dotnet/csharplang/issues/3435) | [list-patterns](https://github.com/dotnet/roslyn/tree/features/list-patterns) | [Merged in 17.1p1](https://github.com/dotnet/roslyn/issues/51289) | [alrz](https://github.com/alrz) | [jcouv](https://github.com/jcouv), [333fred](https://github.com/333fred) | [333fred](https://github.com/333fred) |
+| [Newlines in interpolations](https://github.com/dotnet/csharplang/issues/4935) | main | [Merged in 17.1p1](https://github.com/dotnet/roslyn/issues/57154) | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) | [jcouv](https://github.com/jcouv), TBD | [CyrusNajmabadi](https://github.com/CyrusNajmabadi) |
```

Comment:
>17.1p1

17.1p2?

Reply:
I think 17.1p1 is correct based on the label on the PR: https://github.com/dotnet/roslyn/pull/56853

Reply:
I wouldn't ask if p1 hasn't been already released.

Reply:
@alrz I don't have a way to verify directly (I have 17.1p3 installed...) so I'm inferring from different sources of information. From email notifications I see that the snap for 17.1p1 occurred on Oct 25th before 9pm (was planned after 4pm). The PR for newlines in interpolations was [merged](https://github.com/dotnet/roslyn/pull/56853) on Oct 25th at 1pm.

If you're saying you've tried 17.1p1 and the feature was not there, then I'll change the notes to 17.1p2 just to be safe.

Tagging @CyrusNajmabadi in case he knows something about which preview this went in.

Reply:
You're right, newlines in interpolations is available in 17.1p1. But list patterns are not (unlike what the current iteration says).

Reply:
Corrected list-patterns. Thanks!